Source/RxSwift/count.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  count.swift
+//  RxSwiftExt-iOS
+//
+//  Created by Fred on 06/11/2018.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import Foundation
+import RxSwift
+
+extension Observable {
+    /**
+     Count the number of items emitted by an Observable
+     - seealso: [count operator on reactivex.io](http://reactivex.io/documentation/operators/count.html)
+     - returns: An Observable sequence containing a value that represents how many elements in the specified observable sequence satisfy a condition if provided, else the count of items.
+     */
+    public func count() -> Observable<Int> {
+        return reduce(0, accumulator: { cnt, _ in cnt + 1 })
```

Comment:
Let's get rid of the label and use the trailing-closure syntax here: 

```suggestion
        return reduce(0) { count, _ in count + 1 }
```